ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -393,14 +394,20 @@ private void registerDdlCommandTasks() {
     ddlCommandTasks.put(CreateStream.class, (statement, statementText, properties) -> {
       CreateStreamCommand createStreamCommand =
               new CreateStreamCommand((CreateStream) statement, properties, ksqlEngine.getTopicClient());
-      new DDLCommandExec(ksqlEngine.getMetaStore().clone()).execute(createStreamCommand);
+      DDLCommandResult ddlCommandResult = new DDLCommandExec(ksqlEngine.getMetaStore().clone()).execute(createStreamCommand);
```

Comment:
Looks like this block of code L397 -> 401 could be extracted into a method and used in other places below, i..e, L407 -> 411, L423 -> 428 etc.
Also, is throwing an exception correct here? What do we do when we get an exception from one of these? Would it be better to return something that indicates success/error?

Reply:
Good point, created a method for executing the DDLCommands.
If we have an exception, it will include the information on why the exception happened and will be thrown to upper levels to be shown to the user.